docs/api/hardhat/plugins.md

```diff
@@ -106,19 +113,27 @@ networks: {
 ```
 
 - `version` is a field with the version of the `zkvyper` compiler. Currently not used.
-- `compilerSource` is a field with the compiler source. It can be either `docker` or `binary`. In the former case, the `dockerImage` value with the name of the compiler docker image should also be provided. In the latter case, the compiler binary should be provided in `$PATH`.
-- `compilerPath` is a field with the path to the `zkvyper` binary. If `compilerSource` is `docker`, this field is ignored. By default, the binary in `$PATH` is used.
+- `compilerSource` indicates the compiler source and can be either `docker` or `binary` (recommended). If there's not a compiler binary already installed, the plugin will automatically download it. If `docker` is used, you'd need to run Docker desktop in the background and provide both `dockerImage` and `tag` in the experimental section.
```

Comment:
Can you rephrase it to :
If no compiler binary was installed ..... or If there isn't a compiler binary already installed ....

Reply:
fixed in e28f4343571352dec4b23cfb303616a2f8f497fa